src/promptflow/promptflow/azure/_restclient/flow_service_caller.py

```diff
@@ -675,6 +675,26 @@ def cancel_flow_run(
             **kwargs,
         )
 
+    @_request_wrapper()
+    def resume_bulk_run(
+        self,
+        subscription_id,  # type: str
+        resource_group_name,  # type: str
+        workspace_name,  # type: str
+        body=None,  # type: Optional["_models.ResumeBulkRunRequest"]
+        **kwargs,  # type: Any
+    ):
+        """Cancel a flow run."""
+        headers = self._get_headers()
+        return self.caller.bulk_runs.resume_bulk_run(
```

Comment:
Seems `resume_bulk_run` is not implemented in _bulk_runs_operations.py yet?
I don't understand how does this function work now.

Reply:
Offline synced. that resume_bulk_run is restclient code generated by autorest.